test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,177 @@ Feature: Bind an application to a service
+                              - binding:env:object:secret:username
+                              - binding:env:object:secret:password
+                            - description: Name of the ConfigMap to hold the DB config
+                              displayName: DB Config Map
+                              path: configmap
+                              x-descriptors:
+                              - urn:alm:descriptor:io.kubernetes:ConfigMap
+                              - binding:env:object:configmap:db_host
+                              - binding:env:object:configmap:db_port
+            """
+        * The CRD "backservs.service.example.com" is present
+            """
+            apiVersion: apiextensions.k8s.io/v1beta1
+            kind: CustomResourceDefinition
+            metadata:
+                name: backservs.service.example.com
+            spec:
+                group: service.example.com
+                versions:
+                    - name: v1
+                      served: true
+                      storage: true
+                scope: Namespaced
+                names:
+                    plural: backservs
+                    singular: backserv
+                    kind: Backserv
+                    shortNames:
+                    - bs
+            """
+        * The CR "demo-backserv-cr-2" for kind "Backserv" is present
+            """
+            apiVersion: service.example.com/v1
+            kind: Backserv
+            metadata:
+                name: demo-backserv-cr-2
+            status:
+                secret: csv-demo-secret
+                configmap: csv-demo-cm
+            """
+        * The "configmap" "csv-demo-cm" is present
+            """
+            apiVersion: v1
+            kind: ConfigMap
+            metadata:
+                name: csv-demo-cm
+            data:
+                db_host: 172.72.2.0
+                db_port: "3306"
+            """
+        * The "secret" "csv-demo-secret" is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: csv-demo-secret
+            type: Opaque
+            stringData:
+                username: admin
+                password: secret123
+            """
+        When Service Binding Request is applied to connect the database and the application
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+                name: sbr-csv-secret-cm-descriptors
+            spec:
+                backingServiceSelector:
+                    group: service.example.com
+                    version: v1
+                    kind: Backserv
+                    resourceRef: demo-backserv-cr-2
+            """
+        Then Secret "sbr-csv-secret-cm-descriptors" must contain data from resource type "secret" cr name "csv-demo-secret" and jsonpath "{.data.username}"
```

Comment:
@pratikjagrut The acceptance tests are failing most likely a timing issue because you try to check for a secret that is supposed to be created by the binding, but you do so before that happens... You should probably verify anything about the secret only after the binding is finished, which is ensured by the `.status.condition[]` steps... So put the "Secret must contain data" steps after the SBR conditions. You know that after those steps passed, you are 100% sure that the secret is there.

Could you try this https://github.com/pratikjagrut/service-binding-operator/pull/1 ?

Reply:
https://github.com/redhat-developer/service-binding-operator/pull/666 fixes that

Reply:
Thanx @pmacik. Done